main/routes/all_questions_route.py

```diff
@@ -0,0 +1,16 @@
+from .baseRoutes import *
+
+
+@app.route('/api/v1/questions', methods = ['GET'])
+def get_all_questions():     
+    """This endpoint will fetch all questions """
+    listed_questions = []
+    if questions_list == []:
+        message = {"message":"No questions exist on this platform"}
+        return message["message"], 200
+    else:
+        for question in questions_list:
```

Comment:
If you already have a list you want to return why would you even loop through it again to generate another list?
Just return the questions_list

Reply:
it's returning a dictionary not a list